misc_docs/syntax/extension_embed_raw_expression.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: "embed-raw-expression"
+keywords: ["javascript", "embed", "raw", "expression"]
+name: "%raw"
+summary: "This is the `embed JavaScript expression` extension point."
+category: "extensionpoints"
+---
+
+`%raw` is used to embed JavaScript expressions.
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res
+let canUseCanvas: unit => bool = %raw(`
+  function canUseCanvas() {
+    return !!document.createElement('canvas').getContext;
+  }
+`)
+```
+
+```js
+var canUseCanvas = function canUseCanvas() {
+  return !!document.createElement("canvas").getContext;
+};
+```
+
+</CodeTab>
+
+See `%%raw` for embedding top level blocks of JavaScript code rather than expressions.
```

Comment:
I guess we can directly refer to that as well!

```suggestion
See [`%%raw`](/syntax-lookup#embed-raw-top-level) for embedding top level blocks of JavaScript code rather than expressions.
```

Reply:
Links like this within the Syntax Lookup are not working for me at the moment. Maybe the router is not detecting the URL hash changes, or something like that? What do you think, would it be worth including this now in anticipation of updating the Syntax Lookup routing behaviour?